### PR TITLE
Add PHP set-fingerprint code

### LINKED ISSUE #3035: Missing documentation for SDK fingerprinting for PHP
## Important Details

This an issue with: PHP SDK documentation

## Description

I was trying to figure out how to add addtional fingerprinting onto some of the errors thrown in my PHP application and thus went to: https://docs.sentry.io/platforms/javascript/data-management/event-grouping/sdk-fingerprinting/

### What you expected to happen

I would like to see code snippets that inform me of a correct and idiomatic implementation.

### Possible Solution

Add code snippets similar to https://docs.sentry.io/platforms/javascript/data-management/event-grouping/sdk-fingerprinting/

Also, hi @stayallive 👋

### PR DESCRIPTION
**Pull request by stayallive**
Fixes #3035.

/cc @HazAT

### CODE REVIEW

File: src/includes/set-fingerprint/basic/php.mdx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+```php
+try {
+  // Run code that possibly throws an exception
+} catch (\Throwable $e) {
+  \Sentry\withScope(function (\Sentry\State\Scope $scope) use ($e) {
+    // Group errors together based on the message of the exception being thrown
```
Comment (spawnia) [inline src/includes/set-fingerprint/basic/php.mdx:6]:
```suggestion
    // Group errors together based on a static string
```

File: src/includes/set-fingerprint/database-connection/php.mdx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+```php
+try {
+  // Run code that possibly throws an exception
+} catch (\Throwable $e) {
+  \Sentry\withScope(function (\Sentry\State\Scope $scope) use ($e) {
+    // Group errors together based on the message of the exception being thrown
+    $scope->setFingerprint(['database-connection-error']);
```
Comment (spawnia) [inline src/includes/set-fingerprint/database-connection/php.mdx:7]:
```suggestion
    $scope->setFingerprint(['{{ default }}', $e->getMessage()]);
```

File: src/includes/set-fingerprint/basic/php.mdx (hunk 1 above)
Comment (spawnia) [inline src/includes/set-fingerprint/basic/php.mdx:6]: I still don't get this comment. There is no usage of the exception message whatsoever.

Comment (stayallive) [reply src/includes/set-fingerprint/basic/php.mdx]: 🤦 hope to have fixed it now correctly!

Comment (spawnia) [reply src/includes/set-fingerprint/basic/php.mdx]: Perfextion

**Review by spawnia — APPROVED** [review]

**Review by HazAT — APPROVED** [review]

**Pull request merged**